sympy/series/limits.py

```diff
@@ -104,7 +106,10 @@ def __new__(cls, e, z, z0, dir="+"):
 
         obj = Expr.__new__(cls)
         obj._args = (e, z, z0, dir)
-        return obj
+        if not evaluate:
```

Comment:
I mean you should respect global evaluate context.  Please read documentation in core/evaluate.py, read other methods with evaluate kwarg.